base/errorshow.jl

```diff
@@ -850,10 +850,11 @@ function process_backtrace(t::Vector, limit::Int=typemax(Int); skipC = true)
             code = lkup.linfo
             if code isa MethodInstance
                 def = code.def
-                if def isa Method
-                    if def.name === :kwcall && def.module === Core
-                        continue
-                    end
+                if def isa Method && def.name !== :kwcall && def.sig <: Tuple{typeof(Core.kwcall),Any,Any,Vararg}
```

Comment:
Maybe add a comment why `def.name !== :kwcall`? That's a bit confusing.

Reply:
In theory, it means that we chose a different name for the Method when defining this because it was the kwsorter for something else, but in practice, it is unclear if that is really what it means or not.